tests/test_fkgl.py

```diff
@@ -4,6 +4,7 @@
 from easse.utils.resources import get_orig_sents, get_refs_sents
 
 
+@pytest.mark.skip(reason='TODO: Test broken, need to investigate')
```

Comment:
Isn't that because we detokenized and truecased the test set?
If yes, we could just replace with turkcorpus_test_legacy (and maybe add a new test with turkcorpus_test).